metricbeat/_meta/fields.common.yml

```diff
@@ -49,20 +49,20 @@
           type: scaled_float
           description: Percent CPU used. This value is normalized by the number of CPU cores and it ranges from 0 to 1.
         - name: network.in.bytes
-          type: scaled_float
+          type: long
```

Comment:
Dashboards are ok with this change, right?

Reply:
I don't think they are used in any dashboard yet. But I should change the field types under `aws/ec2` and then check the dashboards 😄